microsoft-edge/webview2/how-to/debug-visual-studio-code.md

```diff
@@ -112,18 +112,18 @@ You might need to attach the debugger to running WebView2 processes.  To do that
 
 Your WebView2 control must open the Chrome Developer Protocol (CDP) port to allow debugging of the WebView2 control.  Your code must be built to ensure that only one WebView2 control has a CDP port open, before starting the debugger.
 
-You will also need to add a new REGKEY `*--remote-debugging-port=9222` under `Computer\HKEY_CURRENT_USER\Software\Policies\Microsoft\Edge\WebView2\AdditionalBrowserArguments` so that the debugger can find the proper port. To add this registry key:
+You will also need to add a new REGKEY `<yourApp.exe> = --remote-debugging-port=9222` under `Computer\HKEY_CURRENT_USER\Software\Policies\Microsoft\Edge\WebView2\AdditionalBrowserArguments` so that the debugger can find the proper port. To add this registry key:
 
 1. Open the Registry Editor by pressing the **Windows logo key** and then searching for **registry editor**. Open the **Registry Editor** application, and then select **Yes** to allow editing.
 
-1. Set the registry key `HKEY_CURRENT_USER\Software\Policies\Microsoft\Edge\WebView2\AdditionalBrowserArguments` equal to `--remote-debugging-port=9222`.
+1. Set the registry key `HKEY_CURRENT_USER\Software\Policies\Microsoft\Edge\WebView2\AdditionalBrowserArguments\<yourApp.exe>` equal to `--remote-debugging-port=9222`.
 
    To do this, in the editor, navigate to `HKEY_CURRENT_USER\Software\Policies\Microsoft\Edge\WebView2\AdditionalBrowserArguments` by clicking on each subfolder under the path.
 
    If this path doesn't exist, navigate to `HKEY_CURRENT_USER\Software\Policies\Microsoft` in the editor, right-click the `Microsoft` folder, select **New**, and then select **Key**.  Enter `Edge` for the name of the new key.  Continue to do this for each subfolder until you have the full path: `HKEY_CURRENT_USER\Software\Policies\Microsoft\Edge\WebView2\AdditionalBrowserArguments`.
 
 1. Right-click the `AdditionalBrowserArguments` folder, select **New**, and then select **String Value**.
-Rename `New Value #1` to `*`.
+Rename `New Value #1` to `<yourApp.exe>`.
 
 1. Right click the **\*** value, and then select **Modify**.  Set the `Value Data` equal to `--remote-debugging-port=9222`.  Verify that the edit window matches the following:
```

Comment:
```suggestion
1. Right click the **<yourApp.exe>** value, and then select **Modify**.  Set the `Value Data` equal to `--remote-debugging-port=9222`.  Verify that the edit window matches the following:
```

Also, I think we should update the screenshots in the following steps to match. I attached them here:

![image](https://github.com/user-attachments/assets/5af9c744-9b9e-4c76-939d-8877c58b89ab)
![image](https://github.com/user-attachments/assets/d5af8399-c441-4947-9197-b1d37048f84c)

Reply:
I'm not sure how to properly update the images. Do either of you know and/or can add that change here?

Reply:
rendered article section:
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/how-to/debug-visual-studio-code?branch=pr-en-us-3240#debug-running-processes

first image:
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/how-to/debug-visual-studio-code-images/set-debugging-port.png

2nd image:
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/how-to/debug-visual-studio-code-images/set-debugging-port-registry-key.png

target local image dir:
C:\Users\v-mihoffman\GitHub\edge-developer\microsoft-edge\webview2\how-to\debug-visual-studio-code-images

quick & maybe lo-res update: I'll DL the above to that dir

Reply:
pushed commit to rewrite reg editor ui steps & created fresh version of those two screenshot pngs
Pls tech review, @champnic ( / @victorhuangwq )